pkg/metrics/diskstats_linux.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright 2015 The Prometheus Authors.
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/openebs/node-disk-manager/cmd/controller"
+	"github.com/openebs/node-disk-manager/cmd/storage/block"
+	"github.com/openebs/node-disk-manager/cmd/types/v1"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	diskType       = "disk"
+	diskSubsystem  = ""
+	diskSectorSize = 512
```

Comment:
ssd's have 4k sector size. We should not assume this to fixed. We should detect this and put the value.